src/v/pandaproxy/api/api-doc/schema_registry_header.json

```diff
@@ -2,11 +2,11 @@
   "swagger": "2.0",
   "info": {
     "title": "Pandaproxy Schema Registry",
-    "version": "1.0.2"
+    "version": "1.0.3"
   },
   "host": "{{Host}}",
   "basePath": "/",
   "schemes": [
     "http"
   ],
-  "paths": {
\ No newline at end of file
+  "paths": {
```

Comment:
This is one of the rare cases where a newline at the end of the file is not desirable.

Reply:
Yeah, I figured as much. I thought I had it right that last time, but I think emacs is being annoying. Will try `nano` or something.

Reply:
It's very rare, and I don't think JSON specifies it, hence nitpick. Oh, I didn't specify nitpick.

Reply:
kk. addressed in [6e2023a](https://github.com/redpanda-data/redpanda/pull/13020/commits/6e2023adbbca27cadff49379466c0d22d1cbc7c4)